File: Nhom-18/my-project/App/screen/Splash.js
import React, {Component} from 'react';
import { View, Text } from 'react-native';

export default class Splash extends Component  {
    render(){
        return (
            <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center', backgroundColor: '#66bb6a', }}>
              <Text style={{color:'white', fontSize:20,}}>HẠNG A1,A2..,B1,B2,C,D,E..</Text>
              <Text style={{color:'white', fontSize:15,}}>ÔN THI GIẤY PHÉP LÁI XE</Text>
            </View>
          );
    }
}

File: 62/flutter_app10/lib/pages/splash.dart
import 'dart:async';
import 'package:flutter/material.dart';
import 'package:flutterapp10/pages/home.dart';
import 'package:flutterapp10/testing/ui/colors.dart';

class splashscreen extends StatefulWidget {
  @override
  _splashscreenState createState() => _splashscreenState();
}

class _splashscreenState extends State<splashscreen> {

  @override
  void initState(){
    super.initState();
    Timer(Duration(seconds: 3), (){
      Navigator.of(context).pushReplacement(MaterialPageRoute(
        builder: (context) => homepage(),
      ));
    });
  }

  // added test yourself
  // and made the text to align at center
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Stack(
        fit: StackFit.expand,
        children: <Widget>[
          Container(
            decoration: new BoxDecoration(
              image: new DecorationImage(
                image: new ExactAssetImage('assets/images/computerr.png',),
                fit: BoxFit.cover,
              ),
            ),
          ),
          Column(
            mainAxisAlignment: MainAxisAlignment.start,
            children: <Widget>[
              Expanded(
                flex: 2,
                child: Container(
                  child: Column(
                    mainAxisAlignment: MainAxisAlignment.center,
                    children: <Widget>[
                      Text(
                        "Học lý thuyết",
                        style: TextStyle(
                          color: Colors.white,
                          fontSize: 50
                        ),
                      )

                    ],
                  ),
                ),
              )
            ],
          )
        ],
      )
//      backgroundColor: Colors.lightGreen,

    );
  }
}